libraries/mbed/targets/hal/TARGET_NORDIC/TARGET_MCU_NRF51822/TARGET_DELTA_DFCM_NNN40/mbed_overrides.c

```diff
@@ -115,9 +118,11 @@ void mbed_sdk_init()
     {// Do nothing.
     }
 	
-	flash_init();
-	
-	//nrf_delay_ms(10);
-	flash_powerDown();
+/* No init flash in this version, 2015/10/27 */
```

Comment:
During merging I had some git freeze, but after next try it continued. Strange is those changes are not shown when I compare changes between branches (https://github.com/mbedmicro/mbed/compare/master...RafalFabich:master). Can I resolve this by closing this pr and creating new one?